modules/otel-send-traces-and-metrics-to-otel-collector-without-sidecar.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 //
-// * otel/otel-using.adoc
+// * otel/otel-sending-traces-and-metrics-to-otel-collector.adoc
```

Comment:
Since a lot of content is moving around in this PR, are there any redirects required? Might want to check this per https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit#bookmark=id.xb255koulgqx

Reply:
Thank you so much for the reminder. I'm not aware of any xrefs that would be affected, but I will grep for them to make sure there aren't any of which I might not be aware.

Reply:
I'll follow up on this in my after-release PR. The release date is next week, so I don't want to risk delaying the release due to this.